sdk/core/core-amqp/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.1.6 (Unreleased)
```

Comment:
We may want to release core-amqp to use new version of core-auth

Reply:
Can we list out the side affects of not doing a core-amqp release with new version of core-auth?

Reply:
We will have two pinned versions of core-tracing, if we think it's acceptable then can skip it this time.

Reply:
That is only when we use core-amqp with other libraries with newer versions of tracing. Not sure how common though.

Reply:
core-auth needs core-tracing only to define it in an interface. It doesnt really get used otherwise. So, we should be good. @xirzec?

Reply:
Since the interfaces are compatible I think it's ok